script/nc_to_mmd

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+"""
+Script to create an MMD xml file from a netcdf file.
+
+License:
+     This file is part of the S-ENDA-Prototype repository (https://github.com/metno/py-mmd-tools).
+     S-ENDA-Prototype is licensed under GPL-3.0 (https://github.com/metno/py-mmd-tools/blob/master/LICENSE)
+
+Usage: nc_to_mmd [-h] -i INPUT -o OUTPUT_DIR
+Example: python nc_to_mmd -i /home/Data/mmd/Input/myfile.nc -o /home/Data/mmd/Output/
```

Comment:
please modify as suggested in the other script by using the `tests/data` directory and some available nc-files